src-qt5/desktop-utils/lumina-textedit/i18n/l-te_sv.ts

```diff
@@ -312,15 +312,16 @@ Vill du stänga den ändå ?
     <message>
         <location filename="../MainUI.cpp" line="546"/>
         <source>Save Changes before closing?</source>
-        <translation type="unfinished"></translation>
+        <translation>Spara ändringar innan du stänger?</translation>
     </message>
     <message>
         <location filename="../MainUI.cpp" line="547"/>
         <source>There are unsaved changes.
 Do you want save them before you close the editor?
 
 %1</source>
-        <translation type="unfinished"></translation>
+        <translation>Det finns osparade ändringar.
+                           Vill du spara dom innan du stänger editorn?</translation>
```

Comment:
```suggestion
                           Vill du spara dom innan du stänger editorn?
                           
                           %1</translation>
```
Ensure the replaceable-string tag is added back in.